app/src/main/java/com/github/se/icebreakrr/ui/sections/Filter.kt

```diff
@@ -0,0 +1,324 @@
+                    textStyle = TextStyle(fontSize = (textFieldHeight.value * textSizeFactor).sp),
+                    isError = ageFromText.isNotEmpty() && (ageFrom == null || ageRangeError))
+                Spacer(modifier = Modifier.width(screenWidth * 0.02f))
+                Text(
+                    " - ",
+                    modifier =
+                        Modifier.padding(horizontal = 8.dp).align(Alignment.CenterVertically),
+                    fontSize = 24.sp,
+                    fontWeight = FontWeight.Bold)
+                Spacer(modifier = Modifier.width(screenWidth * 0.02f))
+                OutlinedTextField(
+                    value = ageToText,
+                    onValueChange = { validateAndUpdateAgeTo(it) },
+                    modifier =
+                        Modifier.height(IntrinsicSize.Min)
+                            .width(textFieldWidth)
+                            .testTag("AgeToTextField")
+                            .onFocusChanged { if (!it.isFocused) validateAgeRange() },
+                    label = { Text("To") },
+                    singleLine = true,
+                    keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
+                    textStyle = TextStyle(fontSize = (textFieldHeight.value * textSizeFactor).sp),
+                    isError = ageToText.isNotEmpty() && (ageTo == null || ageRangeError))
+              }
+
+          val tagSelectorHeight = screenHeight * tagSelectorHeightFactor
+          val tagSelectorWidth = screenWidth * tagSelectorWidthFactor
+
+          Text(
+              "Tags",
+              fontSize = (screenHeight.value * titleFontSizeFactor).sp,
+              modifier = Modifier.padding(vertical = 8.dp).testTag("TagsTitle"))
+          TagSelector(
+              selectedTag = selectedTags,
+              outputTag = availableTags,
+              stringQuery = stringQuery,
+              expanded = expanded,
+              onTagClick = { t ->
+                if (selectedTags.value.any { it.first == t }) {
+                  selectedTags.value = selectedTags.value.filter { it.first != t }
+                }
+              },
+              onStringChanged = {},
+              textColor = MaterialTheme.colorScheme.onSurface,
+              onDropDownItemClicked = { tag ->
+                if (selectedTags.value.none { it.first == tag }) {
+                  availableTags.value
+                      .firstOrNull { it.first == tag }
+                      ?.let { newTag -> selectedTags.value += newTag }
+                }
+              },
+              height = tagSelectorHeight,
+              width = tagSelectorWidth,
+              textSize = (tagSelectorHeight.value * tagSelectorTextSizeFactor).sp)
+          Box(modifier = Modifier.fillMaxWidth().padding(16.dp)) {
+            Button(
+                onClick = { navigationActions.navigateTo(Screen.AROUND_YOU) },
```

Comment:
You can use the funciton navigationActions.goBack() here, it is more optimal i think

Reply:
You're right. Thanks.